mono_repo/lib/src/ci_shared.dart

```diff
@@ -52,11 +52,13 @@ class CIJobEntry {
     required bool includePackage,
     required bool includeStage,
   }) {
+    final flavor = job.flavor == PackageFlavor.dart ? 'Dart' : 'Flutter';
```

Comment:
Probably better to use a switch or read a string out of a map or something? If a new flavor is added at some point this would have to be updated.

Reply:
Yup. Extensions FTW